jobs/ci-kubernetes-e2e-gce-enormous-cluster.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+readonly testinfra="$(dirname "${0}")/.."
+
+### provider-env
+export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://test-container.sandbox.googleapis.com/"
+export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/ci/staging"
+export E2E_MIN_STARTUP_PODS="8"
+export FAIL_ON_GCP_RESOURCE_LEAK="true"
+export KUBERNETES_PROVIDER="gke"
+export ZONE="us-central1-f"
+
+### project-env
+# XXX Not a unique project
+export PROJECT="kubernetes-gce-enormous-cluster"
```

Comment:
FYI I renamed it to the job name here or else it won't pass the unit test. Let me know otherwise.